src/main/webapp/index.html

```diff
@@ -20,6 +20,23 @@ <h1>
     <input type="text" id="node-name" name="search-name" placeholder="Name"></input>
   </div>
   <button onclick="graph.generateGraph()">Get Graph</button>
+
+  <div>
+    <label>Find Node in Graph: </label>
+    <input type="text" id="node-search" placeholder="Node Name"></input>
+    <button id="search-button">Find Node</button>
+    <label id="node-error"></label>
+  </div>
+
+  <div>
+    <label>Find Token in Graph: </label>
+    <input type="text" id="token-search" placeholder="Token Name"></input>
```

Comment:
also when you search by a token that appears in multiple, it only displays the tippy of the first one. do you think it might be better to either display all or none? or say if it's greater than a certain number than display none else all
![image](https://user-images.githubusercontent.com/48307028/88431930-e5bcec00-cdc0-11ea-985d-b74b0c9fb5e9.png)

Reply:
or you could have arrows that navigate to the next node (out of 174) and display the tippy